### DISCUSSION TITLE
Trigger dropdown when clicking selected items

### DISCUSSION THREAD

**Discussion by heatsink99** [orchidjs/tom-select#121] [Q&A]
When using the remove_button and no_active_items plugins, is there a way to trigger the control to dropdown when clicking selected items?  I ask because the the clickable/tap area can be quite small when the selected items fill the control.  And when the clear_button is added, its quite easy to accidentally tap clear when trying to tab at the end of the selected items.

**Comment by oyejorge** [comment]
[Something like this](https://jsfiddle.net/zwe6c0gt/)
